Cargo.toml

```diff
@@ -17,6 +17,7 @@ build = "build.rs"
 libc = "0.2"
 nom = { version = "2", features = ["verbose-errors"] }
 byteorder = "0.5"
+time = "0.1"
```

Comment:
Yep, that's what a I meant.  After thinking more about it, though, what if you changed the Limit struct slightly to be type parameterized, like so:

```rust
pub struct Limit<T> {
    pub lower: Option<T>,
    pub upper: Option<T>,
}
```

Then the Limits struct would become:

pub struct Limits {
    pub max_cpu_time: Limit<Duration>,
    pub max_processes: Limit<usize>,
    ...
}
```

That may be easier to understand/use.